tests/integration/test_sqs.py

```diff
@@ -1298,6 +1300,21 @@ def test_publish_get_delete_message_batch(self, sqs_client, sqs_create_queue):
         )
         assert "Messages" not in confirmation.keys()
 
+    @pytest.mark.aws_validated
+    def test_delete_message_batch_invalid_msg_id(self, sqs_create_queue, sqs_client, snapshot):
+        self._add_error_detail_transformer(snapshot)
+
+        queue_name = f"queue-{short_uid()}"
+        queue_url = sqs_create_queue(QueueName=queue_name)
+
+        invalid_ids = ["", "testLongId" * 10, "invalid:id"]
+
+        for idx, invalid_id in enumerate(invalid_ids):
+            delete_entries = [{"Id": invalid_id, "ReceiptHandle": "testHandle1"}]
+            with pytest.raises(ClientError) as e:
+                sqs_client.delete_message_batch(QueueUrl=queue_url, Entries=delete_entries)
+            snapshot.match(f"error_response_{idx}", e.value.response)
```

Comment:
nit: i guess we could use parameterization here:

```python
    @pytest.mark.parametrize(argnames="invalid_message_id", argvalues=["", "testLongId"*10, "invalid:id"])
    def test_delete_message_batch_invalid_msg_id(self, invalid_message_id, sqs_create_queue, sqs_client, snapshot):
        # ... proceed without loop code
        snapshot.match("error_response", e.value.response)
```

Reply:
nice, much cleaner! 👌